controls/radchartview/populating-with-data/chartdatasource.md

```diff
@@ -239,22 +219,36 @@ Below you'll find a sample where this is demonstrated (the datasource is the sam
 	        return new ChartMinFunction();
 	    }
 	}
-	{{endregion}}
-
-
-
-#### __VB.NET__
+{{endregion}}
 
+#### __[VB.NET] Example 5: Create custom series and overriding its aggregate function__

```

Comment:
Create custom series and override its aggregate function or Creating custom series and overriding its aggregate function